beniget/beniget.py

```diff
@@ -849,6 +849,12 @@ def visit_Subscript(self, node):
 
     visit_Starred = visit_Await
 
+    def visit_NamedExpr(self, node):
+        dnode = self.chains.setdefault(node, Def(node))
+        self.visit(node.target)
+        self.visit(node.value).add_user(dnode)
```

Comment:
I think you should visit `node.value` before `node.target` to correctly handle case like
```
a = 1
(a := a + a)
```

Reply:
Fixed!